src/box.tsx

```diff
@@ -1,38 +1,39 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import {BoxProps} from './types/box-types'
-import {propTypes} from './enhancers'
-import enhanceProps from './enhance-props'
-
-export default class Box extends React.Component<BoxProps, {}> {
-  static displayName = 'Box'
-
-  static propTypes = {
-    ...propTypes,
-    innerRef: PropTypes.func,
-    is: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
-    boxSizing: propTypes.boxSizing
-  }
+import React from "react";
+import PropTypes from "prop-types";
+import { BoxProps } from "./types/box-types";
```

Comment:
We should probably re-export `BoxProps` globally too, if someone wants to use them. In later versions of TS we can use `React.ComponentProps<type Box>` but that may not be suitable always.